src/Business Foundation/App/NoSeries/src/Upgrade/NoSeriesUpgrade.Codeunit.al

```diff
@@ -0,0 +1,21 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Foundation.NoSeries;
+
+codeunit 328 "No. Series Upgrade"
+{
+    Access = Internal;
+    Subtype = Upgrade;
+    InherentEntitlements = X;
```

Comment:
I do not thing these are needed, upgrade always runs with maximum permissions/entitlements.

Reply:
I still believe it's a good custom to have all our objects in either a permission set or with inherent permissions.